tests/test_atomic_create.py

```
@@ -317,3 +317,53 @@ def test_operators_are_warned_once_about_the_degraded_mount(
+        
+        # Use very small poll budget so the test runs quickly
+        monkeypatch.setattr(atomic_io, "_CLAIM_POLL_ATTEMPTS", 2)
+        monkeypatch.setattr(atomic_io, "_CLAIM_POLL_INTERVAL", 0.01)
+        monkeypatch.setattr(os, "link", _no_hardlinks)
+
+        # Now run a loser that will poll the claim
+        # The claim should be seen as live (owner alive, same boot)
+        # The target does not exist (since we haven't written it yet)
+        try:
+            returned = atomic_create_bytes(target, b"loser-bytes")
+            assert False, "Expected OSError from exhausted poll budget"
```

Ruff (0.16.3)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)